scripts/jenkins/jobs-obs/heat-templates/openstack-devstack.yaml

```diff
@@ -0,0 +1,242 @@
+heat_template_version: 2015-10-15
+
+description: |
+  This template builds the OpenStack cleanvm virtual machines for various cloud
+  sources (including a TESTHEAD flag) and images.
+
+  Usage examples:
+
+    # Build from Cloud:OpenStack:Mitaka:Staging on SLE12 SP1
+    heat stack-create -f openstack-cleanvm.yaml -P key_name=myuser -P TESTHEAD=1 myuser-cleanvm
```

Comment:
`heat` command is deprecated for Master, I think we should switch to `openstack` client